test/assets/shipyard_onboard_service.yaml

```diff
@@ -54,7 +54,34 @@ spec:
                 deploymentstrategy: "direct"
             - name: "release"
 
-    - name: "production"
+    - name: "prod-a"
+      sequences:
+        - name: "delivery"
+          triggeredOn:
+            - event: "staging.delivery.finished"
+          tasks:
+            - name: "deployment"
+              properties:
+                deploymentstrategy: "blue_green_service"
+            - name: "release"
+        - name: "rollback"
+          triggeredOn:
+            - event: "production.delivery.finished"
+              selector:
+                match:
+                  result: "fail"
+          tasks:
+            - name: "rollback"
+        - name: "delivery-direct"
+          triggeredOn:
+            - event: "staging.delivery-direct.finished"
```

Comment:
I think this should be `prod-a.delivery-direct.finished`

Reply:
Why has this been staging anyway... Do we have a mistake in our examples?

Reply:
this trigger is correct, since the sequence in `prod-a` should be started after the sequence in staging has finished